front/src/applications/operationalStudies/components/SimulationResults/SpaceCurvesSlopes.js

```diff
@@ -238,14 +237,29 @@ export default function SpaceCurvesSlopes(props) {
       rotate,
       timePosition
     );
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [chart, mustRedraw, positionValues, timePosition]);
 
   useEffect(() => {
     if (chartXGEV) {
       setChart({ ...chart, x: chartXGEV });
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [chartXGEV]);
 
+  useEffect(() => {
+    let timeOutFunctionId;
+    const timeOutResize = () => {
+      clearTimeout(timeOutFunctionId);
```

Comment:
This kind of logic is better hidden inside of a custom hook that we can reuse. The less low level apis we can see the better.

https://usehooks.com/useEventListener/

Reply:
Absolutely, however I won't do it now, @alexandredamiron will refactor and isolate the component soon.